README.md

```diff
@@ -1,10 +1,29 @@
+[![Documentation](https://img.shields.io/readthedocs/nest-simulator?logo=readthedocs)](https://nest-simulator.org/documentation)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/2218/badge)](https://bestpractices.coreinfrastructure.org/projects/2218)
 [![License](http://img.shields.io/:license-GPLv2+-green.svg)](http://www.gnu.org/licenses/gpl-2.0.html)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3605514.svg)](https://doi.org/10.5281/zenodo.3605514)
-[![Latest Version](https://img.shields.io/badge/latest%20version-2.20.0-brightgreen.svg)](https://github.com/nest/nest-simulator/releases/tag/v2.20.0)
-[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/2218/badge)](https://bestpractices.coreinfrastructure.org/projects/2218)
+
+[![Latest release](https://img.shields.io/github/release/nest/nest-simulator.svg?color=brightgreen&label=latest%20release)](https://github.com/nest/nest-simulator/releases)
+[![GitHub contributors](https://img.shields.io/github/contributors/nest/nest-simulator?logo=github)](https://github.com/nest/nest-simulator)
+[![GitHub commit activity](https://img.shields.io/github/commit-activity/y/nest/nest-simulator?color=%23ff6633)](https://github.com/nest/nest-simulator)
+[![GitHub All Releases](https://img.shields.io/github/downloads/nest/nest-simulator/total?logo=github)](https://github.com/nest/nest-simulator/releases)
+
+[![Conda version](https://img.shields.io/conda/vn/conda-forge/nest-simulator.svg?logo=conda-forge&logoColor=white)](https://anaconda.org/conda-forge/nest-simulator)
+[![Conda downloads](https://img.shields.io/conda/dn/conda-forge/nest-simulator?logo=conda-forge&logoColor=white&color=blue)](https://anaconda.org/conda-forge/nest-simulator)
+[![Ubuntu version](https://img.shields.io/badge/ubuntu-v2.20.0%20(PPA)-blue?logo=debian)](https://nest-simulator.readthedocs.io/en/latest/installation/)
+[![Fedora package](https://img.shields.io/fedora/v/nest?logo=fedora)](https://src.fedoraproject.org/rpms/nest)
+[![Homebrew version](https://img.shields.io/homebrew/v/nest.svg?logo=apple)](https://formulae.brew.sh/formula/nest)
+
+[![Virtual applicance](https://img.shields.io/badge/VM-v2.20.0-blue?logo=CodeSandbox)](https://nest-simulator.readthedocs.io/en/latest/download.html#download-livemedia)
+[![Docker Image Version](https://img.shields.io/docker/v/nestsim/nest/2.20.0?label=docker&logo=docker)](https://hub.docker.com/r/nestsim/nest)
+[![Docker Image Size](https://img.shields.io/docker/image-size/nestsim/nest)](https://github.com/nest/nest-docker)
+[![Docker Pulls](https://img.shields.io/docker/pulls/nestsim/nest?label=pulls)](https://hub.docker.com/r/nestsim/nest)
+
+[![YouTube Video Views](https://img.shields.io/youtube/views/K7KXmIv6ROY?style=social)](https://www.youtube.com/results?search_query=nest-simulator+neurons)
+[![Twitter Follow](https://img.shields.io/twitter/follow/nestsimulator?style=social)](https://twitter.com/nestsimulator)
```

Comment:
I'd like plain links without the counts better.

Reply:
@steffengraber, should these social links better appear on the nest-simulator.org (only)?

Reply:
I would be fine with both solutions.

Reply:
> @steffengraber, should these social links better appear on the nest-simulator.org (only)?

I would say yes.